holidays/countries/macau.py

```diff
@@ -0,0 +1,474 @@
+
+            # Restoration of Independence Day.
+            self._add_holiday_dec_1(tr("恢復獨立紀念日"))
+        else:
+            # The Buddha's Birthday.
+            self._add_chinese_birthday_of_buddha(tr("佛誕節"))
+
+            # The day following National Day of the People's Republic of China.
+            self._add_holiday_oct_2(tr("中華人民共和國國慶日翌日"))
+
+            # Macao S.A.R. Establishment Day.
+            self._add_holiday_dec_20(tr("澳門特別行政區成立紀念日"))
+
+    def _populate_mandatory_holidays(self):
+        """
+        Decreto-Lei n.º 101/84/M - Earliest Available Version Online.
+        Decreto-Lei n.º 24/89/M - Added Ching Ming Festival.
+        Lei n.º 8/2000 - Removed Day of Portugal
+                       - Added Macao S.A.R. Establishment Day.
+                       - Moved Mid-Autumn to Day following Mid-Autumn to match Public Holidays.
+        Lei n.º 7/2008 - Consolidated with other laws, reaffirming 2000 Amendment list.
+        """
+        if self._year <= 1984:
+            return None
+
+        # New Year's Day.
+        self._add_new_years_day(tr("元旦"))
+
+        # Chinese New Year's Day.
+        self._add_chinese_new_years_day(tr("農曆正月初一"))
+
+        # The second day of Chinese New Year.
+        self._add_chinese_new_years_day_two(tr("農曆正月初二"))
+
+        # The third day of Chinese New Year.
+        self._add_chinese_new_years_day_three(tr("農曆正月初三"))
+
+        # Labor Day.
+        self._add_labor_day(tr("勞動節"))
+
+        # Double Ninth Festival.
+        self._add_double_ninth_festival(tr("重陽節"))
+
+        # National Day of the People's Republic of China.
+        self._add_holiday_oct_1(tr("中華人民共和國國慶日"))
+
+        # Decreto-Lei n.º 24/89/M - Adds Ching Ming as a Mandatory Holiday.
+        if self._year >= 1989:
+            # Tomb-Sweeping Day.
+            self._add_qingming_festival(tr("清明節"))
+
+        # Lei n.º 8/2000 - Removed Day of Portugal as a Mandatory Holiday.
+        #                - Changed observance from Mid-Autumn to the following day.
+        #                - Adds Macao S.A.R. Establishment Day as a Mandatory Holiday.
+        if self._year <= 1999:
+            # Day of Portugal, Camões, and the Portuguese Communities.
+            self._add_holiday_jun_10(tr("葡國日、賈梅士日暨葡僑日"))
+
+            # Mid-Autumn Festival.
+            self._add_mid_autumn_festival(tr("中秋節"))
+
+        else:
+            # The Day following Mid-Autumn Festival.
+            self._add_mid_autumn_festival_day_two(tr("中秋節翌日"))
+
+            # Macao S.A.R. Establishment Day.
+            self._add_holiday_dec_20(tr("澳門特別行政區成立紀念日"))
+
+    def _populate_government_holidays(self):
+        # While Cross-Checking References are available for from 2005-2025,
+        # SUN in-lieus starts in 2011; SAT-SUN in-lieus starts in 2012.
+        if self._year <= 2004:
+            return None
+
+        dts_observed = set()
+
+        # %s (Afternoon).
+        begin_time_label = self.tr("%s （下午）")
```

Comment:
```suggestion
        begin_time_label = self.tr("%s（下午）")
```
Seems space before U+FF08 is not needed?

Reply:
Fixed, will push the commit in soon once the tests are done 👍